config/database.yml

```diff
@@ -81,6 +86,8 @@ test:
 #
 production:
   <<: *default
-  database: swlcgdb_production
-  username: swlcgdb
-  password: <%= ENV["SWLCGDB_DATABASE_PASSWORD"] %>
+  database: <%= ENV["DATABASE_NAME"] %>
+  username: <%= ENV["DATABASE_USERNAME"] %>
+  password: <%= ENV["DATABASE_PASSWORD"] %>
+  host: <%= ENV["DATABASE_HOST"] %>
+  port: <%= ENV["DATABASE_PORT"] %>
```

Comment:
Add a carriage return to the end of this file as well.

Reply:
+1